docs/Verification_Datasets/datasets/metar_isu.rst

```diff
@@ -0,0 +1,49 @@
+.. _vx-data-metar-isu:
+
+METAR Data
+----------
+
+Description
+  Meteorological Aviation Routine Weather Report (METAR) is a format for observational weather data, which typically comes from airports or permanent observing stations around the world.
+
+  Iowa State University maintains an archive of METAR data:
+  https://mesonet.agron.iastate.edu/request/download.phtml
+
+  Note: METAR data is also available via other sources (e.g., MADIS, NCEP prepbufr files); however, ISU provides a long archive and does not require an account.
+
+Recommended use
+  METAR data are commonly used for near-surface verification of standard meteorological variables, such as temperaures, dew point temperature, wind speed, visibility, and precipitation, and precipitation type.
```

Comment:
Typo: "temperaures" should be "temperatures"
Grammar: should be "precipitation, and precipitation type" (get rid of first "and")